src/main/resources/lib/ssb/cron/cron.ts

```diff
@@ -196,133 +189,140 @@ export function setupCronJobs(): void {
+      clearPartFromPartCache('pifCalculator')
+      clearPartFromPartCache('bkibolCalculator')
+      clearPartFromPartCache('husleieCalculator')
+      clearPartFromPartCache('omStatistikken')
+      clearPartFromPartCache('releasedStatistics')
+      clearPartFromPartCache('upcomingReleases')
+      clearPartFromPartCache('articleList')
+      clearPartFromPartCache('relatedFactPage')
+      clearPartFromPartCache('archiveAllPublications-nb')
+      clearPartFromPartCache('archiveAllPublications-en')
+    },
+    context: cronContext,
+  })
 
-    // Update SDDS tables
-    const updateSDDSTablesCron: string =
-      app.config && app.config['ssb.cron.updateSDDSTables'] ? app.config['ssb.cron.updateSDDSTables'] : '01 09 * * *'
-    schedule({
-      name: 'Update SDDS tables',
-      cron: updateSDDSTablesCron,
-      callback: () => {
-        libScheduleTestLog('updateSDDSCronTest', updateSDDSTablesCron)
-        updateSDDSTables()
-      },
-      context: cronContext,
-    })
-    libScheduleTest({ name: 'updateSDDSCronTest', cron: '01 09 * * *', timeZone: 'Europe/Oslo' }, updateSDDSTablesCron)
+  // Update SDDS tables
+  const updateSDDSTablesCron: string =
+    app.config && app.config['ssb.cron.updateSDDSTables'] ? app.config['ssb.cron.updateSDDSTables'] : '01 09 * * *'
+  schedule({
+    name: 'Update SDDS tables',
+    cron: updateSDDSTablesCron,
+    callback: () => {
+      libScheduleTestLog('updateSDDSCronTest', updateSDDSTablesCron)
+      runOnMasterOnly(updateSDDSTables)
+    },
+    context: cronContext,
+  })
+  // libScheduleTest({ name: 'updateSDDSCronTest', cron: '01 09 * * *', timeZone: 'Europe/Oslo' }, updateSDDSTablesCron)
 
-    const datasetPublishCron: string =
-      app.config && app.config['ssb.cron.publishDataset'] ? app.config['ssb.cron.publishDataset'] : '50 05 * * *'
+  const datasetPublishCron: string =
+    app.config && app.config['ssb.cron.publishDataset'] ? app.config['ssb.cron.publishDataset'] : '50 05 * * *'
 
-    log.info('Run old datasetPublishCron cron-library')
-    // publish dataset cron job
-    schedule({
-      name: 'Dataset publish',
-      cron: datasetPublishCron,
-      callback: () => {
-        libScheduleTestLog('datasetPublishCronTest', datasetPublishCron)
-        publishDataset()
-      },
-      context: cronContext,
-    })
-    libScheduleTest(
-      { name: 'datasetPublishCronTest', cron: '50 07 * * *', timeZone: 'Europe/Oslo' },
-      datasetPublishCron
-    )
+  log.info('Run old datasetPublishCron cron-library')
+  // publish dataset cron job
+  schedule({
+    name: 'Dataset publish',
+    cron: datasetPublishCron,
+    callback: () => {
+      libScheduleTestLog('datasetPublishCronTest', datasetPublishCron)
+      runOnMasterOnly(publishDataset)
+    },
+    context: cronContext,
+  })
+  // libScheduleTest({ name: 'datasetPublishCronTest', cron: '50 07 * * *', timeZone: 'Europe/Oslo' }, datasetPublishCron)
 
+  // Task
+  if (isMaster()) {
```

Comment:
Vurder å inverter denne istedenfor å nøste så mye kode under. 
```
if(!isMaster()) { 
 ...
 return
}

...```

Reply:
Ikke viktig, bare å ignorere dersom det blir litt styrete eller økt risiko for feil.